interview-questions.md

```diff
@@ -9,61 +9,71 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is the difference between a parameter and an argument?
 
   Your answer:
+  - The difference is that parameters are the values listed inside the variable of a given function and the arguements are the injected values into the function's input.
 
   Researched answer:
-
+  - Through my research I learned that the difference between a parameter and an arguement is their own individual values they both represent when interacting within the function. The parameter is the representation of values that defines a function that is malleable for processing after interacting with the arguement and the arguement is the expression that interacts with a new "real" values that will be injected inside the parameter values.
```

Comment:
Parameters are placeholder and arguments are the values actually passed. Very close relationship.